test/Makefile

```
@@ -29,6 +29,10 @@ build:
 	$(GINKGO) build -tags=integration_tests
 	$(QUIET)$(MAKE) -C bpf/
 
+build-darwin:
+	@$(ECHO_GINKGO)$@
+	GOOS=darwin $(GINKGO) build -tags=integration_tests
```

Woops, missed this one before. `GOOS=darwin $(GINGKO) ...` can end up being translated into `GOOS=darwin @ ginkgo ...`, which I think prevents Make from interpreting the `@` (for quiet). Ideally whichever binaries are defined by make variables, those should appear at the start of the line. Maybe we can move the `GOOS` env var to the end of the line?

```suggestion
	$(GINKGO) build -tags=integration_tests GOOS=darwin
```

---

>Maybe we can move the GOOS env var to the end of the line?

That doesn't work. 

```
ginkgo build -tags=integration_tests GOOS=darwin
Found no test suites
For usage instructions:
        ginkgo help
make: *** [build-darwin] Error 1
```

---

Ah okay. In that case, I think I'm the main one affected by the quiet output so maybe we just merge this as-is and I can play around with it later if it's bothering me.